deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -85,6 +85,13 @@ spec:
           - signers
           verbs:
           - approve
+        - apiGroups:
+          - machine.openshift.io
```

Comment:
Not sure why this `apiGroup` is used

Reply:
it was generated as a part of csv generation for the role modification

Reply:
Sorry. I am not sure why the `machine.openshift.io` apiGroup is used in `role.yaml`

Reply:
sorry missed that, corrected it.